src/validations/rules/sap.sch

```diff
@@ -452,6 +474,7 @@
             context="oscal:assessment-plan">
             <sch:assert
                 diagnostics="is-profile-document-diagnostic"
+                fedramp:specific="true"
```

Comment:
This is specific to OSCAL (not FedRAMP). All of the inter-document elements are fundamental OSCAL constructs.